internal/playlist/list.go

```diff
@@ -0,0 +1,30 @@
+package playlist
+
+import (
+	"fmt"
+	"spotify/internal"
+
+	"github.com/spf13/cobra"
+)
+
+func NewListCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"l"},
+		Short:   "list playlists",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			api, err := internal.Authenticate()
+			if err != nil {
+				return err
+			}
```

Comment:
Same thing here... stick everything under this line in a function called `List()` so it can be tested against the mock API